rust/ast/generate_ast.py

```diff
@@ -45,6 +45,11 @@ def to_rust_type(self, ast):
         params_str = ", ".join(p.to_rust_type(ast) for p in self.params)
         if self.name == 'String':
             return "&'alloc str"
+        if self.name == 'Option':
+            if self.params[0].name == 'Box':
+                return "Option<{}>".format(self.params[0].to_rust_type(ast))
+            else:
+                return "Option<{}>".format(params_str)
```

Comment:
I couldn't spot the difference. `params_str` already puts all parameters thorugh `.to_rust_type(ast)`.

Reply:
I mean: to me it looks like these 5 lines don't do anything.

Reply:
yeah, looks like an artifact from some prototyping.
will remove `then` part